Tribler/community/tunnel/hidden_community.py

```diff
@@ -326,10 +345,11 @@ def on_create_e2e(self, messages):
                 relay_circuit.tunnel_data(message.candidate.sock_addr, TUNNEL_PREFIX + message.packet)
             else:
                 self._logger.debug('On create e2e: create rendezvous point')
-                
-                self.create_rendezvous_point(self.hops[message.payload.info_hash],
-                                             lambda rendezvous_point, message=message: 
-                                             self.create_created_e2e(rendezvous_point, message))
+                self.create_rendezvous_point(
```

Comment:
I added some indenting to the lambda thing that's one of the arguments passed to create_rendezvous. Reverted. :wink: